libs/common/src/errors/amendment-not-found.error.ts

```diff
@@ -0,0 +1,11 @@
+import { HttpStatusCode } from 'axios';
+import { ApiError } from './api.error';
+
+export class AmendmentNotFoundError extends ApiError {
```

Comment:
Great idea to add this!

Reply:
Few lines on when this should be called and when it is called will be great please.